variables.tf

```diff
@@ -16,6 +16,15 @@ variable "eks_node_allowed_cidr_blocks" {
 }
 variable "eks_subnet_ids" { type = list(string) }
 variable "node_iam_role_name" {}
```

Comment:
ok so just to see that I get this right

this role in the eks module is used by the node groups
and the role inside the karpenter module (`KarpenterTF-${aws_eks_cluster.eks.id}`) is used by karpenter nodes?